data/utils/convert-arpa.lua

```diff
@@ -64,7 +64,7 @@ assert(ofarpa, string.format('cannot open <%s> for writing', opt.arpadst))
 -- src arpa
 local ifarpa
 if opt.arpasrc:match('%.z$') or opt.arpasrc:match('%.gz$') then
-   ifarpa = io.popen(string.format('zcat %s', opt.arpasrc))
```

Comment:
Could you please move it in a separate commit?

Reply:
Done.